### DISCUSSION TITLE
Training Model

### DISCUSSION THREAD

**Discussion by blessziamah** [coqui-ai/TTS#4024] [General Q&A]
Is it possible for me to train a new model with a dataset that contains only voice recordings? No transcript

**Comment by samarasimhapeyala** [comment]
Training a text-to-speech model typically requires both audio recordings and corresponding transcripts (text) to learn the mapping between spoken sounds and written language. If you only have voice recordings without transcripts, you would face a significant challenge because the model needs the text to understand what to synthesize. You can use ASR models like [whisper](https://github.com/openai/whisper) to generate transcripts for your audios.

Please mark it answered if  your doubt is resolved.
